### PR TITLE
Fix .chat chat when muted

### PR DESCRIPTION
**Pull request by vhiperdev**
Fix for issues

https://github.com/azerothcore/mod-world-chat/issues/7

### CODE REVIEW

**Comment by FrancescoBorzi** [comment]
Travis build failed

**Comment by BarbzYHOOL** [comment]
thank you @vhiperdev

@andoys test this plz

File: src/WorldChat.cpp [hunk 1]
```diff
@@ -112,6 +112,9 @@ class World_Chat : public CommandScript
         Player* player = pChat->GetSession()->GetPlayer();
         uint32 guid = player->GetGUID();
 
+        if (!pChat->GetSession()->GetPlayer()->CanSpeak())
```
Comment (wizzymore) [inline src/WorldChat.cpp]: You should use player->CanSpeak() instead

**Comment by wizzymore** [comment]
You can look at this commit: https://github.com/wizzymore/mod_world_chat/commit/43b4ba7a3e4826ccde09be2c9f23375398e31ee8 for a better idea

File: src/WorldChat.cpp [hunk 2]
```diff
@@ -112,6 +112,9 @@ class World_Chat : public CommandScript
...
+            return false;
```
Comment (wizzymore) [inline src/WorldChat.cpp]: This should return true

**Comment by wizzymore** [comment]
Because of https://github.com/azerothcore/mod-world-chat/pull/9 we can close this

Thank you @vhiperdev for all your job!

**Pull request closed without merging**